website/docs/utilities/fixupx.md

```diff
@@ -0,0 +1,30 @@
+---
+title: FixupX Reminder
```

Comment:
I wonder if a title like "Replace X (Twitter) links with FixupX" would make this more discoverable for folks searching. What do you think?

Reply:
I guess the question is what is the intent of this CC, a reminder with a fixer added on or a fixer with a reminder added on? I agree with Joe that the latter would be more appealing, but maybe I'm biased because that's what I'd prefer on my server. Imo if you have a system on your server appending fixupx links then it seems excessive to be pressuring your members to change their links manually before sending. Just clogs up the chat I feel. Have you considered making the reminder part of the CC a config value off by default? I like the attention given to make the plurality correct :)